impl/src/main/java/impl/QuestionResourceImpl.java

```diff
@@ -41,15 +43,25 @@ public QuestionResourceImpl(QuestionDao questionDao) {
     @Override
     public Observable<Question> getQuestionBySlackThreadId(String slackThreadId) {
         return this.questionDao.getQuestionBySlackThreadId(slackThreadId).switchIfEmpty(
-            exception(() -> new WebException(HttpResponseStatus.NOT_FOUND, "not_found")));
+            exception(() -> new WebException(HttpResponseStatus.NOT_FOUND, "not found")));
     }
 
 	@Override
 	public Observable<Question> getQuestionById(long questionId) {
       return this.questionDao.getQuestionById(questionId).switchIfEmpty(
-        exception(() -> new WebException(HttpResponseStatus.NOT_FOUND, "not_found")));
+        exception(() -> new WebException(HttpResponseStatus.NOT_FOUND, "not found")));
 	}
 
+    @Override
+    public Observable<List<Question>> getLatestQuestion(Integer limit) {
+        if (limit == null) {
+            limit = 10;
+        }
+        return this.questionDao.getLatestQuestions(limit).toList().onErrorResumeNext(e ->
+            error(new WebException(HttpResponseStatus.INTERNAL_SERVER_ERROR, "failed to get latest questions", e))
```

Comment:
This field is for an error code and not an error message, so I think it needs a bit of reformatting. Same thing with the "not found" once you changed further up.